### DISCUSSION TITLE
Change message language targeting full screen

### DISCUSSION THREAD

**Discussion by lily2brain** [jspsych/jsPsych#1784] [Q&A] [answered]
Hi guys, my experiment is run in the "PORTUGUÊS" language of Brazil, and I would like to know if it is possible to change the language of the message by directing the full screen "The experiment will switch to full screen mode when you press the button below" to make it easier for participants! I await an answer, thank you! ;)

**Comment by jodeleeuw** [comment, marked as answer]
Yes, see the `message` parameter in the documentation: https://www.jspsych.org/plugins/jspsych-fullscreen/

**Reply by lily2brain** [reply]
Thank you for the help! ;)
